en/docs/develop/apis/association-rest-api.md

```diff
@@ -7,7 +7,7 @@ template: templates/swagger.html
 ??? Note "Click for instructions"
     Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server. 
     
-       1.  Click on **Authorize** button and provide desired values for authentication. 
+       1.  Click **Authorize** button and provide the desired values for authentication. 
```

Comment:
```suggestion
       1.  Click **Authorize** and provide the desired values for authentication. 
```

Reply:
Please change in the other docs as well.

Reply:
fixed